app/src/main/java/org/simple/clinic/scanid/IndiaNHIDGender.kt

```diff
@@ -0,0 +1,39 @@
+package org.simple.clinic.scanid
+
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import org.simple.clinic.patient.Gender
+
+enum class IndiaNHIDGender {
+  MALE, FEMALE, UNKNOWN;
+
+  companion object {
+    fun fromIndiaNHIDToGender(value: IndiaNHIDGender): Gender {
+      return when (value) {
+        MALE -> Gender.Male
+        FEMALE -> Gender.Female
+        else -> Gender.Unknown("U")
+      }
+    }
```

Comment:
What I meant was to add this factory function to the `Gender` type and call it `fromIndiaNHIDGender`. `IndiaNHIDGender` should not be responsible for converting itself to the `Gender` type, it should be the other way around.